rollup/app/lwc/rollupForceRecalculation/rollupForceRecalculation.html

```diff
@@ -1,186 +1,188 @@
-            onchange={handleComboChange}
-            options={rollupMetadataOptions}
-            value={selectedMetadata}
-          ></lightning-combobox>
-          <template if:true={selectedMetadataCMDTRecords}>
-            <lightning-datatable
-              class="slds-m-top_large"
-              columns={cmdtColumns}
-              data-id="datatable"
-              data={selectedMetadataCMDTRecords}
-              key-field="DeveloperName"
-              max-row-selection={maxRowSelection}
-              onrowselection={handleRowSelect}
-              show-row-number-column
-            >
-            </lightning-datatable>
-          </template>
-        </template>
-        <template if:false={isCMDTRecalc}>
-          <lightning-input
-            class="slds-col slds-form-element slds-form-element_horizontal"
-            data-id="CalcItem__c"
-            label="Child Object SObject API Name"
-            name="CalcItem__c"
-            oncommit={handleChange}
-            required
-            type="text"
-          >
-          </lightning-input>
-          <lightning-input
-            class="slds-col slds-form-element slds-form-element_horizontal"
-            data-id="RollupFieldOnCalcItem__c"
-            label="Child Object Calc Field"
-            name="RollupFieldOnCalcItem__c"
-            oncommit={handleChange}
-            required
-            type="text"
-          >
-          </lightning-input>
-          <lightning-input
-            class="slds-col slds-form-element slds-form-element_horizontal"
-            data-id="LookupFieldOnCalcItem__c"
-            label="Child Object Lookup Field"
-            name="LookupFieldOnCalcItem__c"
-            oncommit={handleChange}
-            required
-            type="text"
-          >
-          </lightning-input>
-          <lightning-input
-            class="slds-col slds-form-element slds-form-element_horizontal"
-            data-id="LookupObject__c"
-            label="Rollup Object API Name"
-            name="LookupObject__c"
-            oncommit={handleChange}
-            required
-            type="text"
-          >
-          </lightning-input>
-          <lightning-input
-            class="slds-col slds-form-element slds-form-element_horizontal"
-            data-id="RollupFieldOnLookupObject__c"
-            label="Rollup Object Calc Field"
-            name="RollupFieldOnLookupObject__c"
-            oncommit={handleChange}
-            required
-            type="text"
-          >
-          </lightning-input>
-          <lightning-input
-            class="slds-col slds-form-element slds-form-element_horizontal"
-            data-id="LookupFieldOnLookupObject__c"
-            label="Rollup Object Lookup Field"
-            name="LookupFieldOnLookupObject__c"
-            oncommit={handleChange}
-            required
-            type="text"
-          >
-          </lightning-input>
-          <div>
+    <section class="slds-is-relative">
```

Comment:
@jongpie this whole diff is just a painful indentation diff caused by adding this section. the only actual changes are here and on line 185

Reply:
everyone's favorite - minor changes, huge diff